uno.platform.uno-calculator.yaml

```diff
@@ -0,0 +1,86 @@
+app-id: uno.platform.uno-calculator
+runtime: org.gnome.Platform
+runtime-version: '3.28'
```

Comment:
this runtime has been EOL for YEARS.

Reply:
3.28 was what the Snap uses but I've updated to 46 and it still seems to work